src/providers/LocalVideoProvider/docs/LocalVideoProvider.stories.mdx

```diff
@@ -2,7 +2,13 @@
 
 # LocalVideoProvider
 
-The `LocalVideoProvider` provides the local video status, video tile id associated with the local user's video and a function to toggle their video.
+The `LocalVideoProvider` provides video tile id associated with the local user's video, the local video status, a function to set the value of `isVideoEnabled` and a function to toggle their video.
+
+The `LocalVideoProvider` also provides the preview video stauts and a function to set the value of `isPreviewVideoEnabled`.
+
+LocalVideo and PreviewVideo use a same video stream from the selected camera.
+
+`setIsVideoEnabled` and `setIsPreviewVideoEnabled` is only used to manage the status of the `isVideoEnabled` and `isPreviewVideoEnabled`, they can not change the status of the camera.
```

Comment:
Should we just say, to start stop a video preview check `PreviewVideo` documentation, and to start stop user's local video use `toggleVideo` function from the `LocalVideoProvider`?